README.md

```diff
@@ -1,66 +1,55 @@
-# Pure
+# pure [![Fish Shell Version](https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square)](http://fishshell.com)
```

Comment:
@edouard-lopez It feels like screaming at people man! Hahaha. I usually just leave the project title and a brief explanation right below it and I honestly think it is sufficient. Simple and effective. Mind if we leave it that way?